docs/transform/custom_transformers.ipynb

```diff
@@ -0,0 +1,491 @@
+{
```

Comment:
<p>nit: </p><p>"Primitives don't necessarily support everything that could go into a <code>cirq.TransformContext</code> object" </p><p> </p><p>The reason we don't directly take a context object in primitives is:</p><p>a) We want users to be able to call the transformer primitives without necessarily creating a transformer that expects a transformer context (for example, if the transformation that they want to do can be expressed in just a single call to <code style="font-weight: 400; letter-spacing: 0.01em;">cirq.map_operations(circuit, custom_map_func)</code>.  </p><p>b) Transformer primitives don't support automated logging, like the cirq transformers do. </p><p> </p><p>Can we reword the sentence to better reflect this? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5414/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>